cocotb/_build_libs.py

```diff
@@ -149,6 +150,11 @@ def _extra_link_args(lib_name=None, rpath=None):
         args += ["-Wl,-install_name,@rpath/%s.so" % lib_name]
     if rpath is not None:
         args += ["-Wl,-rpath,%s" % rpath]
+
+    # allow 3bit cross compilation
+    if platform.architecture()[0] == "32bit":
+        args += ["-m32"]
```

Comment:
Doesn't python handle this for us? It wouldn't make any sense for it not to.

Reply:
Unfortunately not. Python assumes gcc is also 32bit. I have checked.